From: Sunil Mushran <sunil.mushran@oracle.com>
Subject: ocfs2: Add xattr mount option in ocfs2_show_options()
Patch-mainline: 2.6.28?
References: FATE302067 

Patch adds check for [no]user_xattr in ocfs2_show_options() that completes
the list of all mount options.

Signed-off-by: Sunil Mushran <sunil.mushran@oracle.com>
Signed-off-by: Mark Fasheh <mfasheh@suse.com>
---
 fs/ocfs2/super.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/fs/ocfs2/super.c b/fs/ocfs2/super.c
index 2173169..d789c53 100644
--- a/fs/ocfs2/super.c
+++ b/fs/ocfs2/super.c
@@ -991,6 +991,11 @@ static int ocfs2_show_options(struct seq_file *s, struct vfsmount *mnt)
 		seq_printf(s, ",cluster_stack=%.*s", OCFS2_STACK_LABEL_LEN,
 			   osb->osb_cluster_stack);
 
+	if (opts & OCFS2_MOUNT_NOUSERXATTR)
+		seq_printf(s, ",nouser_xattr");
+	else
+		seq_printf(s, ",user_xattr");
+
 	return 0;
 }
 
-- 
1.5.4.5

